data/views/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,34 @@
+{
+
+    "friendlyName": "<%=friendlyName%>",
+    "injectableName": "<%=injectableName%>",
+    <% if (hasOwnProperty("options")) { %>
+    "options": <%- JSON.stringify(options) %>,
+    <% }%>
+    "tasks":[
+    <% tasks.forEach(function (value, i, arr){ %>
+    {
+    "label": "<%=value.label%>",
+    <% if ( value.waitOn) { %>
+    "waitOn": <%- JSON.stringify(value.waitOn) %>,
+    <% } %>
+
+    <% if ( value.taskDefinition ) { %> 
```

Comment:
That's why I opened the PR for template unit-testing: https://github.com/RackHD/on-http/pull/427